src/modules/user/user.entity.ts

```diff
@@ -0,0 +1,5 @@
+export class User {
```

Comment:
Why would you need to export an entity? Prisma client automatically generates the appropriate types. Please take a look at the prisma/client documentation.